src/PackageManager/Utilities/PackageManagerCefHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CefSharp;
+using Dynamo.ViewModels;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using static Dynamo.PackageManager.PackageDownloadHandle;
```

Comment:
`using static` keyword is not supported by C# 4.5 and our build system is still using .NET 4.5 compiler so for the moment you need to stick to 4.5 convention.